docker/app-init.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# todo: replace prod with $1 and pass in from docker-compose.yml from APP_ENV
+
+cd /var/www/html && \
+    composer run-script db:setup:prod && \
+    composer run-script db:migrate:prod
```

Comment:
`db:setup:prod` calls the script `src/Utils/MigrationHelper.php -> doSetup()`, which should run all of the pending migrations already ... the `db:migrate:prod` might not be necessary.